modules/cassandra/src/main/java/org/testcontainers/containers/CassandraContainer.java

```diff
@@ -152,28 +169,32 @@ public String getPassword() {
 
     /**
      * Get configured Cluster
-     *
+     * <p>
      * Can be used to obtain connections to Cassandra in the container
      */
     public Cluster getCluster() {
-        return getCluster(this, enableJmxReporting);
+        return getCluster(this, this.enableJmxReporting, authenticationEnabled, userName, password);
     }
 
-    public static Cluster getCluster(ContainerState containerState, boolean enableJmxReporting) {
+    public static Cluster getCluster(ContainerState containerState, boolean enableJmxReporting, boolean authenticationEnabled, String userName, String password) {
```

Comment:
This is a breaking change since the `getCluster(ContainerState,boolean)` method is public.

Also, the method looks simple enough and we should encourage users to use the builder directly (with `containerState.getContainerIpAddress()` / `containerState.getMappedPort(CQL_PORT)` as host/port values), so I would not expose this

Alternatively, we could return `Cluster.Builder`.

Reply:
Changed the method signature back to the original.